src/de/mod.rs

```diff
@@ -387,6 +396,72 @@ where
         Self::new(reader)
     }
 
+    /// Set the maximum number of events that could be skipped during deserialization
+    /// of sequences.
+    ///
+    /// If `<element>` contains more than specified nested elements, `#text` or
+    /// CDATA nodes, then [`DeError::TooManyEvents`] will be returned during
+    /// deserialization of sequence field (any type that uses [`deserialize_seq`]
+    /// for the deserialization, for example, `Vec<T>`).
+    ///
+    /// This method can be used to prevent a [DoS] attack and infinity memory
+    /// consumption when parse a very large or even an infinity XML to a sequence
+    /// field.
+    ///
+    /// It is strongly recommended to set limit to some value when you parse data
+    /// from untrusted source. You should choose a value that you typical XMLs
+    /// can have _between_ different elements that corresponds to the same sequence.
+    ///
+    /// # Examples
+    ///
+    /// Let's imagine, that we deserialize such structure:
+    /// ```
+    /// struct List {
+    ///   item: Vec<()>,
+    /// }
+    /// ```
+    ///
+    /// The XML that we try to parse look like this:
+    /// ```xml
+    /// <any-name>
+    ///   <item/>
+    ///   <!-- Bufferisation starts at this point -->
+    ///   <another-item>
+    ///     <some-element>with text</some-element>
+    ///     <yet-another-element/>
+    ///   </another-item>
+    ///   <!-- Buffer will be emptied at this point; was buffered 7 events -->
```

Comment:
```suggestion
    ///   <!-- Buffer will be emptied at this point; 7 events will be drained -->
```

Reply:
It seems that suggested change will change an attention point slightly. I try to emphasize, that buffer must have size enough to store at least 7 events (which reader may unexpect, because from his point of view it may seem that only one element need to be skipped and calculations for possible buffer size can be totally wrong if follow that logic. The real size should be at least 7 times bigger), and after that change is seems that sentence mostly gives information that you get exactly 7 events. Maybe I'm wrong and that change does not change the sence... please tell me your opinion.

Reply:
You could reword it to "7 events were buffered" in that case